cdap-ui/app/cdap/components/SpotlightSearch/SpotlightModal/SpotlightModalHeader.js

```diff
@@ -43,10 +43,14 @@ require('./SpotlightModal.scss');
        <ModalHeader>
          <span className="float-xs-left">
            {
-
-             T.translate('features.SpotlightSearch.SpotlightModal.headerSearchResults', {
-               query: this.props.query
-             })
+             this.props.tag ?
```

Comment:
I don't think we need this check. Spotlight search is not being used for anything other than `shows list of entities matching a tags` as of now. The `query` message was for the spotlight search when used for searching entities.